integration-tests/test_integration/client-integration.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
```

Comment:
Shouldn't this be 2020?

Reply:
Copy paste error, does it need to be cleaned up across the board?

Reply:
It's a "file gets touched, update it" kind of situation.  Sometimes it gets missed.